<launch>

  <node pkg="icr" name="grasp_server" type="grasp_server" output="screen">
        <remap from="set_object" to="/grasp_server/set_object"/>
        <param name="tf_prefix" type="string" value="/srh/position" />  
     <rosparam file="$(find icr)/config/grasp_server_config.yaml" command="load"/>
  </node>


</launch>
